File: app/src/main/java/ru/vtb/ppcd/example/repository/RequestRepository.java
package ru.vtb.ppcd.example.repository;

import ru.vtb.ppcd.example.entity.RequestEntity;
import java.util.UUID;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;


@Repository
public interface RequestRepository extends CrudRepository<RequestEntity, UUID> {
}

File: ms-ppcd-example-app/src/main/kotlin/ru/vtb/ppcd/example/repository/RequestRepository.kt
package org.example.ru.vtb.ppcd.example.repository

import org.example.ru.vtb.ppcd.example.entity.RequestEntity
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository
import java.util.UUID

@Repository
interface RequestRepository : CrudRepository<RequestEntity, UUID>